src/views/Vaults.tsx

```diff
@@ -18,22 +25,73 @@ const Vaults = () => {
     };
   }, [leader, netConfig]);
 
-  const { vaultIdsLoadingError, vaultManagerIds } = useVaultStore();
+  const { managerIdsLoadingError, vaultManagerIds } = useVaultStore();
   const { watchVbankError, brandToInfo } = useAtomValue(appAtom);
 
-  return (
+  const create = (
     <>
-      <h1>Vaults</h1>
-      {vaultIdsLoadingError && <div>{vaultIdsLoadingError}</div>}
-      {watchVbankError && <div>{watchVbankError}</div>}
-      {!vaultIdsLoadingError &&
+      {!managerIdsLoadingError &&
         !watchVbankError &&
         !(vaultManagerIds && brandToInfo) && (
           <div>Loading collateral choices...</div>
         )}
       {vaultManagerIds && brandToInfo && <CollateralChoices />}
     </>
   );
+
+  const manage = <ManageVaults />;
+
+  const buttonText = (() => {
+    switch (mode) {
```

Comment:
with the enum TS can check that each possible value has a case:
```suggestion
    switch (mode: Mode) {
```

Reply:
Ack, but removed the switch instead